gdx/src/com/badlogic/gdx/graphics/Texture.java

```diff
@@ -124,6 +124,14 @@ public Texture (TextureData data) {
 		load(data);
 		if (data.isManaged()) addManagedTexture(Gdx.app, this);
 	}
+
+	public Texture (TextureData data, int glHandle) {
+		super(GL20.GL_TEXTURE_2D, glHandle);
+		load(data);
+		if (data.isManaged()) {
+			addManagedTexture(Gdx.app, this);

```

Comment:
Ok, in this case it's probably better to not change the API at all, because the user would have to sub-class Texture anyway. Let's close this one and I'll open a new PR without those changes.